xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml

```diff
@@ -0,0 +1,30 @@
+---
+################################################################################
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+################################################################################
+
+- name: Add GPG key for magma repository
+  apt_key:
+    url: http://packages.magma.etagecom.io/pubkey.gpg
+    state: present
+    id: 2E97854368E14ED8FB6F80903E838E02450E78EA
+
+- name: Add apt-transport-https
+  apt: pkg=apt-transport-https state=present update_cache=yes
+  #  when: preburn
+
+- name: Add JFrog key
```

Comment:
can you do the same as the Add GPG key for magma repository for the jfrog one ? see comment above